src/main/mainWindow.js

```diff
@@ -43,7 +40,23 @@ function getValidWindowPosition(state) {
   return state;
 }
 
+function handleScreenResize(screen, browserWindow) {
+  screen.on('display-metrics-changed', () => {
```

Comment:
You're right. On Windows, `display-metrics-changed` is emitted after `display-removed`.